src/Sylius/Component/Core/Promotion/Checker/Rule/CustomerGroupRuleChecker.php

```diff
@@ -39,7 +39,11 @@ public function isEligible(PromotionSubjectInterface $subject, array $configurat
             return false;
         }
 
-        if ($configuration['groups'] === $customer->getGroup()->getId()) {
+        if (is_null($customer->getGroup())) {
+            return false;
+        }
+
+        if ($configuration['group'] === $customer->getGroup()->getId()) {
```

Comment:
We should use `code`, same as we do in `TaxonRuleChecker` (can be done in a separate PR).